playbooks/roles/edx_django_service/templates/edx/app/app/app.sh.j2

```diff
@@ -6,7 +6,7 @@
 
 {% set executable = edx_django_service_venv_bin + '/gunicorn' %}
 
-{% if COMMON_ENABLE_NEWRELIC_APP %}
+{% if COMMON_ENABLE_NEWRELIC_APP and edx_django_service_enable_newrelic_app  %}
```

Comment:
Maybe consider changing these conditionals to just use the role's var (and default it to `COMMON_ENABLE_NEWRELIC_APP` rather than to true.)

```suggestion
{% if edx_django_service_enable_newrelic_app %}
```